src/scenes/ProgressReports/Detail/WorkflowCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Card, CardActions, StepperProps, Typography } from '@mui/material';
+import { ProgressReportStatus as Status } from '~/api/schema/schema.graphql';
+import { RelativeDateTime } from '~/components/Formatters';
+import { ButtonLink } from '~/components/Routing';
+import { ProgressReportDetailFragment } from './ProgressReportDetail.graphql';
+import { StatusStepper } from './StatusStepper';
+
+type WorkflowCardProps = {
+  report?: ProgressReportDetailFragment;
+} & StepperProps;
```

Comment:
This doesn't match where you spread the rest of the props.
```suggestion
} & CardProps;
```
```suggestion
} & StyleProps;
```

Reply:
Here, are you suggesting I choose one or use both of them?

Reply:
One of them.

Reply:
Not sure how this didn't get fixed the first time either. I did have to un-commit it a commit I made and I must have made a mistake in that process.